### ISSUE TITLE
Add option to import a csv when creating a new compendium

### ISSUE THREAD

**Issue by jonepatr** [xdy/twodsix-foundryvtt#253]
**Is your feature request related to a problem? Please describe.**
No, but adding a larger number of entities into a compendium is somewhat inconvenient

**Describe the solution you'd like**
When creating a compendium, there being a field to either upload a csv or providing a link to a csv. This csv will then be parsed to see that it contains the correct columns for the given entity and the user will be asked if they want to import N number of objects.

**Describe alternatives you've considered**
An interface with an editable table could be shown where the user can enter the data manually. I suppose this could be used  as an alternative to ask if the user wants to import N objects, and instead present them in an editable table. However, I think that using other software for compiling  a csv (e.g. Excel) is better, and here only focus on the import of the data file.

**Additional context**
None.

**Comment by jonepatr** [comment]
I have been looking into this a bit. How should we handle the import of "items" or "actors"? Since there are many types of items (and actors) with various fields a csv is a bit inflexible in this regard. Only allowing to upload a certain kind of item is also a bit restrictive. I guess JSON would be a better format since it can handle this better, but json is even less accessible to non-developers than csv is.

**Comment by xdy** [comment]
Yeah, you pretty much have the choice of either:
* Limiting it to the types you have programmed in
* Build a generic import tool that parses the json to figure out how the csv:s should look.

I've built a couple of tools like the latter option over the years (though, not for forge obviously). They're fun to build, but tend to be brittle. (At least the ones I've made. :)
If you do decide to build such a tool, I'd suggest you turn it into a module suitable for any system instead.
Might want to take a look at https://github.com/SvenWerlen/fvtt-data-toolbox I just remembered. It has one set up a json file to describe the csv.
Had forgotten about it, but I did use it earlier, some remnants of that remain in https://github.com/xdy/twodsix-foundryvtt/tree/master/src/compendium-imports
